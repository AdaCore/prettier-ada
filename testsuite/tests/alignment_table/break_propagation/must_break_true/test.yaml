title: Table_Alignment Must_Break propagation
driver: document_formatter
description: |
  Test that Aligment_Table Must_Break propagates to rows.
