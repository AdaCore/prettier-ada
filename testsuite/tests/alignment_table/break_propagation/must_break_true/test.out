with GNATCOLL.Traces;
use GNATCOLL.Traces;
with Prettier_Ada.Documents;          use Prettier_Ada.Documents;
with Prettier_Ada.Documents.Builders; use Prettier_Ada.Documents.Builders;
